### PR TITLE
sync to OIM master, oimce not reserved, inf/nan dup check update

### PR DESCRIPTION
**Pull request by hermfischer-wf**
#### Reason for change
sync to OIM master branch, oimce no longer reserved, inf/nan dup check updated

#### Description of change
see above

#### Steps to Test
run OIM conf tests

**review**:
@Arelle/arelle

### CODE REVIEW

File: arelle/plugin/loadFromOIM.py [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "xbrl": nsOims,
     "xs": (XbrlConst.xsd,),
     "enum2": XbrlConst.enum2s,
-    "oimce": nsOimCes,
+    # "oimce": nsOimCes,
```
Comment (sagesmith-wf) [inline arelle/plugin/loadFromOIM.py:99]: Remove?

**Review by sagesmith-wf — APPROVED** [review]
Congratulations on your first contribution Herm!

**Review by derekgengenbacher-wf — DISMISSED** [review]

File: arelle/plugin/loadFromOIM.py (hunk 1 above)
Comment (derekgengenbacher-wf) [reply arelle/plugin/loadFromOIM.py:99]: This causes a couple of the conformance suite tests to fail.

Comment (derekgengenbacher-wf) [reply arelle/plugin/loadFromOIM.py:99]: This causes a couple of the conformance suite tests to fail.

**Comment by aaroncameron-wk** [comment]
Quick rebase so this can run against new CI

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**